proto/execution_stats.proto

```diff
@@ -87,6 +87,15 @@ message Execution {
   // doesn't.
   build.bazel.remote.execution.v2.Digest action_result_digest = 9;
 
+  // A digest used to retrieve the
+  // [build.bazel.remote.execution.v2.ExecuteResponse] produced by the executor
+  // for the latest attempt of this execution.
+  //
+  // This digest can be used to fetch a specially formatted ActionResult from
```

Comment:
Kind of ambivalent here, but did you consider putting this in the CAS instead as the hashed(executeResponse) minus this field?

Reply:
Yes but then we have to store it somewhere. I am hesitant to add more columns to the Executions table but if it seems clearer that way, we can do that instead.